examples/.env.example

```diff
@@ -1,8 +1,9 @@
-DEPLOYER_PRIVATE_KEY="S12srNEAvZrTb9pktYeuePpuM4taW5dfmWAZYtdqyETWTBspkoT1"
-RECEIVER_PRIVATE_KEY="S1ykLaxXyMnJoaWLYds8UntqKTamZ4vcxrZ1fdToR8WpWEpk3FC"
+DEPLOYER_SECRET_KEY="S12srNEAvZrTb9pktYeuePpuM4taW5dfmWAZYtdqyETWTBspkoT1"
+RECEIVER_SECRET_KEY="S1ykLaxXyMnJoaWLYds8UntqKTamZ4vcxrZ1fdToR8WpWEpk3FC"
 
 JSON_RPC_URL_PUBLIC=https://test.massa.net/api/v2
 JSON_RPC_URL_PRIVATE=https://test.massa.net/api/v2
 
+# Local node
 #JSON_RPC_URL_PUBLIC=http://127.0.0.1:33035
 #JSON_RPC_URL_PRIVATE=http://127.0.0.1:33034
```

Comment:
is it needed ? If yes, can you use and if instead of comments as proposed in a previous CR? if not can you clean it ?

Reply:
Cleaned it, we don't need it anymore